caom2-ui-server/src/main/resources/META-INF/resources/observation.jsp

```diff
@@ -47,62 +40,74 @@ This JSP page renders an HTML table with the Observation metadata.
       <td><%= SS.toString(obs.metaRelease) %>
       </td>
     </tr>
-    <tr>
-      <td>sequenceNumber</td>
-      <td><%= SS.toString(obs.sequenceNumber) %>
+    <tr >
+      <td>metaReadGroups</td>
+      <td><%= SS.serializeURISet(obs.getMetaReadGroups()) %>
       </td>
     </tr>
     <tr class="even">
-      <td>algorithm</td>
-      <td><%= SS.toString(obs.getAlgorithm()) %>
+      <td>sequenceNumber</td>
+      <td><%= SS.toString(obs.sequenceNumber) %>
       </td>
     </tr>
     <tr>
-      <td>members</td>
-      <td><%= SS.toMemberString(request.getContextPath(), obs, request.getParameter("ID")) %>
-      </td>
-    </tr>
-    <tr class="even">
       <td>type</td>
       <td><%= SS.toString(obs.type) %>
       </td>
     </tr>
-    <tr>
+    <tr class="even">
       <td>intent</td>
       <td><%= SS.toString(obs.intent) %>
       </td>
     </tr>
     <tr class="even">
-      <td>proposal</td>
-      <td><%= SS.toString(obs.proposal) %>
+      <td>members</td>
+      <td><%= SS.toMemberString(request.getContextPath(), obs, request.getParameter("ID")) %>
       </td>
     </tr>
     <tr>
-      <td>requirements</td>
-      <td><%= SS.toString(obs.requirements) %>
+      <td>algorithm</td>
+      <td><%= SS.toString(obs.getAlgorithm()) %>
       </td>
     </tr>
-    <tr class="even">
+    <tr>
       <td>telescope</td>
       <td><%= SS.toString(obs.telescope) %>
       </td>
     </tr>
-    <tr>
+    <tr class="even">
       <td>instrument</td>
       <td><%= SS.toString(obs.instrument) %>
       </td>
     </tr>
-    <tr class="even">
+    <tr>
+      <td>environment</td>
+      <td><%= SS.toString(obs.environment) %>
+      </td>
+    </tr>
+    <tr>
+      <td>proposal</td>
+      <td><%= SS.toString(obs.proposal) %>
+      </td>
+    </tr>
+    <tr>
       <td>target</td>
       <td><%= SS.toString(obs.target) %>
       </td>
     </tr>
-    <tr>
-      <td>environment</td>
-      <td><%= SS.toString(obs.environment) %>
+    <tr class="even">
+      <td>targetPosition</td>
+      <td><%= SS.toString(obs.targetPosition) %>
+      </td>
+    </tr>
+    <tr class="even">
```

Comment:
Two `even` coloured rows here.  I thought there was a counter that did this automatically.  It may be better to add something that increments with each row and checks for even (`i %2 == 0 ? "even" : ""`) just to avoid this headache unless it's a lot of work.

Reply:
I've found in the displays this attribute is somehow ignored or overridden by something else. I've straightened this one out at the least.